src/device_info.sh

```diff
@@ -341,6 +345,69 @@ function get_desktop_environment()
   device_info_data['desktop_environment']="$formatted_de"
 }
 
+# This function populates kernel variables from the device_info_data
+# variable.
+#
+# @target Target machine
+function get_kernel_info()
+{
+  local target="$1"
+  local flag="$2"
+  local cmd_name
+  local cmd_version
+  local cmd_release
+  local cmd_machine
+  local kernel_name
+  local kernel_version
+  local kernel_release
+  local kernel_machine_type
+
+  cmd_name="uname --kernel-name"
+  cmd_release="uname --kernel-release"
+  cmd_version="uname --kernel-version"
+  cmd_machine="uname --machine"
```

Comment:
Use single quotes.

Reply:
You need to make this change in this commit (first).